ITKImageProcessingFilters/ImportAxioVisionV4Montage.cpp

```diff
@@ -544,7 +591,8 @@ void ImportAxioVisionV4Montage::generateCache(QDomElement& root)
       }
       bound.MetaData = metaAm;
     }
-
+    d_ptr->m_MaxCol = std::max(bound.Col, d_ptr->m_MaxCol);
+    d_ptr->m_MaxRow = std::max(bound.Row, d_ptr->m_MaxRow);
```

Comment:
Update the cache with the max Column & Row. To be used later in error checking and montage information strings